lib/osf-components/addon/components/osf-navbar/home-links/template.hbs

```diff
@@ -17,9 +17,15 @@
     </a>
 </li>
 <li {{action onLinkClick}}>
-    {{#link-to 'support' click=(action 'click' 'link' 'Navbar - navbar.support' target=analytics) }}
-        {{t 'navbar.support'}}
-    {{/link-to}}
+    {{#if onInstitutions}}
+        <a href='http://help.osf.io/m/institutions' onclick={{action 'click' 'link' 'Navbar - navbar.support' target=analytics}}>
+            {{t 'navbar.support'}}
+        </a>
+    {{else}}
+        {{#link-to 'support' click=(action 'click' 'link' 'Navbar - navbar.support' target=analytics) }}
+            {{t 'navbar.support'}}
+        {{/link-to}}
+    {{/if}}
```

Comment:
We should handle contextual support in a more general way as this will certainly not be the only case. Let's discuss.